packages/web/src/AdminRoutesWithBackButton.js

```diff
@@ -0,0 +1,183 @@
+    },
+    {
+        path: 'members/add',
+        element: <AddMember />,
+    },
+    {
+        path: 'jobs',
+        element: <RenderJobs />,
+    },
+    {
+        path: 'jobs/add',
+        element: <AddJobs />,
+    },
+    {
+        path: 'jobs/actions',
+        element: <JobActions />,
+    },
+    {
+        path: 'mentors',
+        element: <RenderMentors />,
+    },
+    {
+        path: 'mentors/actions',
+        element: <MentorsActions />,
+    },
+    {
+        path: 'mentors/add',
+        element: <AddMentors />,
+    },
+    {
+        path: 'jury',
+        element: <RenderJury />,
+    },
+    {
+        path: 'jury/add',
+        element: <AddJury />,
+    },
+    {
+        path: 'jury/actions',
+        element: <JuryActions />,
+    },
+    {
+        path: 'sponsors',
+        element: <RenderSponsors />,
+    },
+    {
+        path: 'sponsors/add',
+        element: <AddSponsors />,
+    },
+    {
+        path: 'sponsors/actions',
+        element: <SponsorsActions />,
+    },
+    {
+        path: 'partners',
+        element: <RenderPartners />,
+    },
+    {
+        path: 'partners/add',
+        element: <AddPartners />,
+    },
+    {
+        path: 'partners/actions',
+        element: <PartnersActions />,
+    },
+    {
+        path: 'hackaton/teams',
+        element: <RenderTeams />,
+    },
+    {
+        path: 'hackaton/teams/actions',
+        element: <TeamActions />,
+    },
+    {
+        path: 'hackaton/teams/add',
+        element: <AddNewTeam />,
+    },
+    {
+        path: 'hackaton/teams/members',
+        element: <RenderTeamMembers />,
+    },
+    {
+        path: 'hackaton/teams/members/add',
+        element: <AddTeamMember />,
+    },
+    {
+        path: 'hackaton/teams/members/actions',
+        element: <TeamMemberActions />,
+    },
+    {
+        path: 'hackaton/noteamparticipants',
+        element: <RenderNoTeamParticipants />,
+    },
+    {
+        path: 'hackathon/noteamparticipants/add',
+        element: <AddNoTeamParticipant />,
+    },
+    {
+        path: 'hackathon/noteamparticipants/actions',
+        element: <NoTeamParticipantsActions />,
+    },
+    {
+        path: 's3',
+        element: <S3Panel />,
+    },
+    {
+        path: 's3/objects',
+        element: <RenderStorageObjects />,
+    },
+    {
+        path: 'shortener',
+        element: <UrlsTable />,
+    },
+    {
+        path: 'fswitches',
+        element: <RenderSwitches />,
+    },
+];
+const AdminRoutesWithBackButton = () => (
+    <Routes>
+        {adminRoutes.map(route => (
+            <Route
+                path={route.path}
+                element={<AdminContainerWithButton component={route.element} path={route.path} />}
+                key={route.path}
+            />
+        ))}
+    </Routes>
+);
+const AdminContainerWithButton = ({ component, path }) => {
+    var position = 'middle';
+    console.log(path);
+    if (path == 'shortener') position = 'top';
```

Comment:
Avoid using var. Use `const` instead and a ternary operator.

```javascript
const position = path === 'shortener' ? 'top' : 'middle'
```

Reply:
Let's add a prop called `positionButtonOnTop` to the button. This way the button can be reused in other components which can't provide a path.

```javascript
<BackBtn positionButtonOnTop={path === 'shortener'}/>

...

const className = positionButtonOnTop ? "top" : "middle";
...
<Button className={className}/>
```